firestore-bigquery-export/functions/package.json

```diff
@@ -13,7 +13,7 @@
   "author": "Jan Wyszynski <wyszynski@google.com>",
   "license": "Apache-2.0",
   "dependencies": {
-    "@firebaseextensions/firestore-bigquery-change-tracker": "^1.1.10",
+    "@firebaseextensions/firestore-bigquery-change-tracker": "^1.1.12",
```

Comment:
Does .11 already exist on NPM?

Reply:
It does...

![image](https://user-images.githubusercontent.com/2060661/115553089-e9749500-a2a4-11eb-813c-6e192193b04e.png)


But BQ doesn't seem to have been updated to the published version. This new version at `.12` reflects changes made in the previous month, so would also include the missing updates from published version `.11`